### DISCUSSION TITLE
How to get tabs to show everywhere

### DISCUSSION THREAD

**Discussion by Nic-Vladimir** [lukas-reineke/indent-blankline.nvim#957] [Q&A] [answered]
How should you configure the plugin to get tabs indicators like this:
![image](https://github.com/user-attachments/assets/8366c43c-1541-4742-af32-49c030e0d8d4)
without errors:
.../share/nvim/lazy/indent-blankline.nvim/lua/ibl/utils.lua:26: char: expected indent.char to have a display width of 0 or 1, got → . Info: '→ ' has a display width of 2

If I use just the arrow by itself I only get tabs to show between text not from the start of the line:
![image](https://github.com/user-attachments/assets/19ca19eb-a0d3-4729-9ed9-dbbf4a7e1193)

current config:
![image](https://github.com/user-attachments/assets/aad3d866-e838-4a1a-a650-5f588901d922)

as you can see I use the modular approach for the lazy package manager

**Comment by lukas-reineke** [comment, marked as answer]
Please take a look at `:help ibl.config.indent.tab_char`

**Reply by Nic-Vladimir** [reply]
Thank you!
